File: Go/euler1.go
// Project Euler Problem 1
// http://projecteuler.net/problem=1

package main

import (
	"fmt"
	"time"
)

func euler() {
	sum := 0
	for i := 1; i < 1000; i++ {
		if i%3 == 0 || i%5 == 0 {
			sum += i
		}
	}
	fmt.Println(sum)
}

func main() {
	t1 := time.Now()
	euler()
	fmt.Println("Elapsed time:", time.Since(t1))
}

File: Python/euler1.py
# Project Euler Problem 1
# https://projecteuler.net/problem=1

from euler import run

def euler():
    result = 0
    for i in range(1, 1000):
        if i % 3 == 0 or i % 5 == 0:
            result += i
    print result

run(euler)
